### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by minhajrahman23** [community/community#32122] [GitHub Education]
I am a student of "University of the People". I am just admitted. The University is fully online based. They provide me with just UoPeople ID: C1704919. How will I get Student Developer Pack, which proves I submit? Please advise me.

![Screenshot_695](https://user-images.githubusercontent.com/95987308/188620951-0b3f0d8d-8448-4d9b-aa61-e17f27b096e9.png)

**Comment by SupaMic** [comment]
Hi @minhajrahman23, 
   contact student.services@uopeople.edu and your Program Advisor (I recommend at the same time on the same email request) to get your University of the People student email address and temporary password and then login to [Office 365](https://www.office.com/) to access this email account as well as the other MS tools.  You may need to choose "Sign in with a different account" from the top right profile button if you already have access under another account.
   
![image](https://github.com/community/community/assets/970239/b5ac487c-28cd-46e5-b293-888517030ccb)

You'll also have to add this email address, once you can verify it, to your Github profile/account settings to use it for validation.

**Reply by ashleyleslie1** [reply]
Github forces me to use my camera and hold that document in front of the camera and it never recognizes the document. its impossible to verify the student status on github. I don't understand why you cant just verify your student status with an email confirmation that you would receive to your edu email account...

**Reply by ducle-uopeople** [reply]
Same issue

**Reply by alihusnainarshad** [reply]
Same Issue

**Reply by samus-aran** [reply]
@alihusnainarshad 👋  Try not to reply to older discussions. We encourage you to use our search bar before posting to find if a similar and more recent discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding! I'll be locking this discussion now.
